### PR TITLE
Change Task ID to directory path

### PR DESCRIPTION
**Pull request by lucpeterson**
Changes celery task ID to be the directory path where the step is being done. This can be used to build other query tools

### CODE REVIEW

**Comment by lucpeterson** [comment]
The two changes are to capture both "sample" tasks and "non sample" tasks

File: CHANGELOG.md
Comment (bgunnar5) [inline CHANGELOG.md:9]: Let's just move this to the 1.12.2b1 release notes as I'm going to wait to release it until this PR and #484 are merged.

**Review by bgunnar5 — APPROVED** [review]
Once the conflicts are resolved this looks good to merge.

**Comment by bgunnar5** [comment]
Need to fix issue where latest changes weren't pulled for some reason. This is a reminder to not merge this branch until that's fixed

**Comment by lucpeterson** [comment]
> Need to fix issue where latest changes weren't pulled for some reason. This is a reminder to not merge this branch until that's fixed

Ok fixed! Merge whenever!

**Pull request merged**
